sdk/core/azure-core/src/cryptography/sha_hash.cpp

```diff
@@ -26,6 +26,7 @@ namespace {
 
 enum class SHASize
 {
+  SHA1,
```

Comment:
isn't sha1 obsolete ? it's known to be broken. Do we have some security person who could weigh in ?

Reply:
A security person like me (I made my bones at Microsoft doing security stuff :))?

SHA1 is not permitted for secure hashing purposes. However, it is *required* for certain protocols and APIs as long as hash collisions aren't relevant. For instance, both the WebSockets protocol and the JsonWebKey file format require SHA1. As does the .Net `X509Certificate2.Thumbprint` API.